### ISSUE TITLE
Google Play Store rejection - malware/uploading contacts

### ISSUE THREAD

**Issue by danielnordh** [lukasgit/flutter_contacts#236]
We use contacts locally on the device for a legitimate reason, and DO NOT upload them to any server but keep getting rejected. We have our Privacy policy linked both from the store listing, the sign up page in the app and the settings page. Yet, we get robotic rejection emails claiming that:

`During review, we found that your app violates the Malware policy. We don’t allow apps with any code that could put a user, a user’s data, or a device at risk. You can read through the Malware policy page and review the User Data policy for more details.`

`For example, your app is uploading users' Contact List information to [our api url] without posting a privacy policy in both the designated field in the Play Developer Console and from within the Play distributed app itself.`

I get the feeling that some automated code tests that Google run is triggering 'something', as this is a completely legitimate use of an official API and we are complying with the Privacy policy part. Have been trying to resolve this for two weeks now without success, super frustrating.

Have other people experienced this and if so how did you get past the reviews?
(We use v0.4.6)

**Comment by paulblakely** [comment]
I just had my app removed this morning for similar reasons, and we also absolutely do not upload the contacts from the device.

In our case, it's for an "invite" form, where someone can enter an email, or choose a contact, and we read the email from that. As exepcted, the appeal to Google is going nowhere so far..

@danielnordh Did you ever get a resolution to this, or do I have to remove this plugin?

**Comment by danielnordh** [comment]
I did eventually get the app approved, but not in a way that made sense.
After trying to communicate with the Google review people (by using their dispute system) and getting nowhere, I submitted a new version WITHOUT the contacts package. This release was approved.
I then submitted a new release WITH the contacts package back in. This was also approved.
No idea why this worked, but it did for me. I'd say this is on Google.
